Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN GOARCH=$TARGETARCH make compile
 RUN mkdir -p output
 
 # Create final image from ubi + built binary
-FROM --platform=$TARGETPLATFORM registry.access.redhat.com/ubi9/ubi:9.3
+FROM --platform=$TARGETPLATFORM registry.access.redhat.com/ubi9/ubi:9.4
```

Comment:
guys, is there a reason here to use the "full" ubi and not the minimal like we do elsewhere ? Is it that we need more tooling in the image?

Reply:
I just tried building running e2e using `ubi-minimal:9.4` it shows no issue I don't think we need the full image here you switched in this PR